gf-webapp-admin/src/component/rooms/room/apartmentType/ApartmentType.tsx

```diff
@@ -0,0 +1,47 @@
+import React, {useState} from "react";
+import {IRoom} from "../../../../model/room";
+import {saveRoom} from "../../../../service/room";
+import {ROOM_TYPES} from "../../../../utils/consts";
+import {EditBtn} from "../editBtn/EditBtn";
+
+
+export const ApartmentsType = ({room}:{room:IRoom}) => {
+
+  const [type, setType] = useState(room.apartmentsType);
+  const [isDisabled, setIsDisabled] = useState(true);
+  const roomTypes = ROOM_TYPES;
+
+  const saveHandler = async () => {
+    setIsDisabled(true);
+    room.apartmentsType = type;
+    saveRoom(room);
+  };
+
+  const disabledHandler = () => {
```

Comment:
Wrong naming, it's hard understand what is disabled and why.

Reply:
Naming has been changed